sounddevice.py

```diff
@@ -2266,9 +2266,27 @@ def reset(self):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
 
-
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors. 
+
+    Attributes
+    ----------
+    paerrorcode : int or None
```

Comment:
Wouldn't `code` be enough?
`PortAudioError.paerrorcode` seems to be repeating itself.